test/test_pexcept.c

```diff
@@ -0,0 +1,71 @@
+/***************************************************************************
+ * Copyright (C) 2016 - 2022 Patricio Bonsembiante. All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ***************************************************************************/
+
+#include "unity.h"
+#include "putils/pexcept.h"
+
+void setUp(void) { }
+
+void tearDown(void) { }
+
+static PEXCEPT_T _throw_an_exception(void) {
+  return 666;
```

Comment:
This should not return, but instead `throw` the exception! 
```suggestion
  throw(666);
```